wolfcrypt/src/tfm.c

```diff
@@ -5735,8 +5746,13 @@ int mp_radix_size (mp_int *a, int radix, int *size)
         return FP_MEM;
 #endif
 
+/* Appease compiler for configurations known to need it.
+**   CONFIG_COMPILER_OPTIMIZATION_PERF: Espressif -O2 */
+#if defined(CONFIG_COMPILER_OPTIMIZATION_PERF)
+    fp_init(t); /* Init t to appease some compilers before call to get a copy. */
```

Comment:
Same here. I'd prefer to see a PR where these are always initialized.

Reply:
Agreed. See latest code changes.